tools/tasks/webpack-dev-server.js

```diff
@@ -12,29 +13,39 @@ let wpds = null; // Hold a reference to Webpack Dev Server when it is created
  *  - HotModuleReplacementPlugin
  *  - Added entry points
  */
-module.exports = function startWebpackDevServer(webpackConfig, localhost, devServerConfig) {
+module.exports = function startWebpackDevServer(webpackConfig, devServerConfig) {
   return (cb) => {
     const localWebpackConfig = webpackConfig;
+    const {
+      entry: { 'pattern-lab': plEntry }, // ./source/pattern-lab.js
+      output: { publicPath }, // /temp/
+    } = webpackConfig;
```

Comment:
Is this the magic that handles the path issues when served from within a container?